Packs/IBMGuardium/Integrations/IBMSecurityGuardium/IBMSecurityGuardium.py

```diff
@@ -0,0 +1,558 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+import urllib3
+import hashlib
+import traceback
+from typing import Any
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+""" CONSTANTS """
+
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
```

Comment:
Format mismatch could cause issues. The code works because dateparser is used, but the constant is misleading